app/jobs/intake_pdf_job.rb

```diff
@@ -0,0 +1,6 @@
+class IntakePdfJob < ApplicationJob
+  def perform(intake_id, filename)
```

Comment:
When Alex and I were doing some DB cleaning last tax season, we noticed that jobs store the params in the db so that they can be run. When they fail, the entire intake would get saved as an attribute on the job. This is why it is usually more standard to pass IDs into jobs instead of the full object.